search/intrinsic/intrinsic.md

```diff
@@ -0,0 +1,23 @@
+# Intrinsic
+
+The `intrinsic` module attaches enumerated values that were created at ingest time to a given search. By default, when using the text or raw renderers, all intrinsic enumerated values are attached to the search. When using any other render module, such as table, the intrinsic module must be used. 
```

Comment:
"Attaches" makes it sound like we're creating them at search time, when in fact we're just... telling the indexer to make sure it grabs them when it reads the entries from disk. I know that "attach" is technically correct but it sorta makes this sound too much like `enrich`.